packages/ui/src/components/DataStory/clients/WorkspacesApi.ts

```diff
@@ -0,0 +1,14 @@
+export interface Tree {
+  path: string;
+  type: 'file' | 'folder';
+  content?: string;
+  children?: Tree[];
+}
+
+export interface WorkspacesApi {
+  get: ({ path }) => Promise<Tree>
+  create: ({ path, tree }: { path: string, tree: Tree }) => Promise<Tree>;
```

Comment:
Is it possible to create multiple files and directories at once?

Reply:
Yes. However, its unclear if/how an UI can do that. I think UI will be able to create trees consisting of:
1. empty diagram
2. empty folder